grails-app/views/deliveryNote/print.gsp

```diff
@@ -9,7 +9,7 @@
     <g:set var="entityName" value="${warehouse.message(code: 'requisition.label', default: 'Requisition')}"/>
     <title><warehouse:message code="default.show.label" args="[entityName]"/></title>
     <link rel="stylesheet" href="${resource(dir:'css',file:'buttons.css')}" type="text/css" media="all" />
-    <g:javascript library="jquery" plugin="jquery" />
+    <g:jquery />
```

Comment:
We can just explicitly include the jquery library we want to use into the print.gsp layout (grails-app/views/layouts/print.gsp), assuming the use of jquery-3.3.1 doesn't break anything.

So replace this 
```
<g:javascript library="jquery" plugin="jquery" />
```
with this
```
    <script src="https://code.jquery.com/jquery-3.3.1.slim.min.js" integrity="sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo" crossorigin="anonymous"></script>
```

Reply:
@jmiranda could you elaborate on this, because I’m confused? 
I’m exactly doing this, but assigned it to a „variable” (a custom tag), so that if we ever need to bump up the version, we’ll just have to change it in the taglib, not in every file